task/coverity-availability-check/0.1/coverity-availability-check.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: sast-coverity-check
+spec:
+  description: >-
+    This task performs needed checks in order to use Coverity image in the pipeline. It will check for a Coverity license secret and an authentication secret for pulling the image.
+  results:
+    - description: Tekton task result output.
+      name: TASK_OUTPUT
```

Comment:
I think the result should be named as `TEST_OUTPUT` like other tasks and it can be used to be checked if the result is pass.

Reply:
Renamed to `TEST_OTUPUT`